src/sql/workbench/parts/query/browser/queryActions.ts

```diff
@@ -552,6 +556,10 @@ export class ToggleSqlCmdModeAction extends QueryTaskbarAction {
 		// set query options
 		let queryoptions: QueryExecutionOptions = { options: new Map<string, any>() };
 		queryoptions.options['isSqlCmdMode'] = toSqlCmdState;
+		if (!this.editor.input) {
+			console.error('editor input was null');
+			return Promise.resolve(null);
```

Comment:
Should just be Promise.resolve() - this method returns Promise<void>

Reply:
Following the flow of the code again here:

https://github.com/microsoft/azuredatastudio/blob/f4c2e28262f5cf81c811ecb99fbc09718fc7703d/src/sql/workbench/parts/query/browser/queryActions.ts#L567